charts/linkerd2/values.yaml

```diff
@@ -92,6 +92,13 @@ global:
   linkerdNamespaceLabel: linkerd.io/is-control-plane
   workloadNamespaceLabel: linkerd.io/workload-ns
 
+  #
+  ## Define your image pull secrets for private docker registries like the following
+  #imagePullSecrets:
```

Comment:
In this comment, please clarify that this is for adding the ImagePullSecret to each of the control plane service accounts (and not adding the ImagePullSecret to each Pod).